submarine-server/server-submitter/submitter-yarn/src/main/java/org/apache/submarine/server/submitter/yarn/YarnUtils.java

```diff
@@ -46,129 +44,61 @@
   private static final Log LOG = LogFactory.getLog(YarnUtils.class);
 
   public static Configuration tonyConfFromClientContext(
-          ParametersHolder parameters) throws YarnException, ParseException {
+          RunJobParameters parameters, Framework framework) throws YarnException, ParseException {
     Configuration tonyConf = new Configuration();
     // Add tony.xml for configuration.
     tonyConf.addResource(Constants.TONY_XML);
     tonyConf.setStrings(TonyConfigurationKeys.FRAMEWORK_NAME,
-            parameters.getFramework().getValue());
+            framework.getValue());
     tonyConf.setStrings(TonyConfigurationKeys.APPLICATION_NAME,
-            parameters.getParameters().getName());
-    tonyConf.setStrings(
-        TonyConfigurationKeys.getInstancesKey(Constants.WORKER_JOB_NAME),
-            parameters.getOptionValue(CliConstants.N_WORKERS));
-    if (parameters.getOptionValue(CliConstants.N_PS) != null) {
-      tonyConf.setStrings(
-              TonyConfigurationKeys.getInstancesKey(Constants.PS_JOB_NAME),
-              parameters.getOptionValue(CliConstants.N_PS));
-    }
-    // Resources for PS & Worker
-    if (parameters.getOptionValue(CliConstants.PS_RES) != null) {
```

Comment:
The method of parameters.getOptionValue works for both cli arguments and yaml configuration.
PR #161 supports this method parameters.getOptionValue for grpc. Maybe we don't need to modify it.

Reply:
@yuanzac Thanks for the review.
> The method of parameters.getOptionValue works for both cli arguments and yaml configuration.

In YarnUtils `parameters.getOptionValue` will parse cli and yaml, and the write correspond conf to tony-xml, but when use pysubmarine client we won't pass `CommandLineProto` or yaml conf to rpc server. We pass `TensorFlowRunJobParameterProto` or `PyTorchRunJobParameterProto` , so that if we use `parameters.getOptionValue` , it can't find any parameter.

Reply:
Got you~
With this modification, does it still work when yaml is used to submit a job?
I just want to make sure that function is not affected.

Reply:
Yes, I just try to submit job with YAML. it works well.

Reply:
@pingsutw,
Thanks. you mean for both java client and python client, a job can be submitted with YAML, right?